connect-example/src/main/java/com/stripe/android/connect/example/ui/accountloader/EmbeddedComponentLoader.kt

```diff
@@ -0,0 +1,147 @@
+package com.stripe.android.connect.example.ui.accountloader
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.ModalBottomSheetLayout
+import androidx.compose.material.ModalBottomSheetValue
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.material.rememberModalBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.stripe.android.connect.EmbeddedComponentManager
+import com.stripe.android.connect.PrivateBetaConnectSDK
+import com.stripe.android.connect.example.R
+import com.stripe.android.connect.example.ui.common.Async
+import com.stripe.android.connect.example.ui.common.Fail
+import com.stripe.android.connect.example.ui.common.Loading
+import com.stripe.android.connect.example.ui.common.Success
+import com.stripe.android.connect.example.ui.common.Uninitialized
+import com.stripe.android.connect.example.ui.settings.SettingsView
+import kotlinx.coroutines.launch
+
+/**
+ * Manages the UI for loading an [EmbeddedComponentManager], including error states.
+ * Calls [content] with a successful manager instance when available.
+ */
+@OptIn(PrivateBetaConnectSDK::class)
+@Composable
+fun EmbeddedComponentLoader(
```

Comment:
I think this need "Manager" in the name, otherwise it sounds like it's loading an embedded component
```suggestion
fun EmbeddedComponentManagerLoader(
```